modules/components/pages/processors/benchmark.adoc

```diff
@@ -0,0 +1,60 @@
+= benchmark
+// tag::single-source[]
+:type: processor
+:page-beta: true
+:categories: ["Utility"]
+
+component_type_dropdown::[]
+
+Logs throughput statistics for messages that pass through this processor, and provides a summary of those statistics over the lifetime of the processor.
+
+ifndef::env-cloud[]
+Introduced in version 4.40.0.
+endif::[]
+
+```yml
+# Configuration fields, showing default values
+label: ""
+benchmark:
+  interval: 5s
+  count_bytes: true
+```
+
+== Throughput statistics
+
+This output logs the following rolling statistics at a <<interval,configurable interval>> to help you to understand the current performance of your pipeline:
+
+- The number of messages processed per second.
+- The number of bytes of messages processed per second (optional).
```

Comment:
"bytes of messages" feels weird to me here. I would just say "bytes processed per second"

Reply:
Updated.